dist/js/homepage.js

```diff
@@ -5136,10 +5136,12 @@ baltimoreCounty.constants = (function constants() {
 	var baltCoGo = {
 		urls: {
 			api: {
-				geocodeServer: '//bcgis.baltimorecountymd.gov/arcgis/rest/services/Geocoders/CompositeGeocode_CS/GeocodeServer',
+				geocodeServer:
+                    '//bcgis.baltimorecountymd.gov/arcgis/rest/services/Geocoders/CompositeGeocode_CS/GeocodeServer',
 				createReport: rootUrl + '/api/baltcogo/createreport',
 				getReport: rootUrl + '/api/citysourced/getreport',
-				getReportLatLng: rootUrl + '/api/citysourced/getreportsbylatlng',
+				getReportLatLng:
+                    rootUrl + '/api/citysourced/getreportsbylatlng',
```

Comment:
indent?

Reply:
@jsmith176 notice this is in the dist folder, so we are not controlling the format

Reply:
That's right, I missed that. This is good to go.